week3/homework/src/app.js

```diff
@@ -0,0 +1,56 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const utilities = require('./utilities');
+
+/* before dealing with the endpoints:  parse the request body */
+app.use(express.json());
+
+// ( 0 )
```

Comment:
if these are clarifying for you it's fine during your development, but once you done removing them will give more space for others and less lines to read, image a server with 100 route?